src/inputs/index.js

```diff
@@ -36,6 +36,12 @@ export default class Input {
         this.store.dispatch(ActionCreators.updateBundle(bundle));
         return this.api.getInputs(bundle)
         .then(inputs => {
+            if (!Array.isArray(inputs)) {
```

Comment:
Our `this.api.xxx` methods really should throw errors/reject when the response comes back with an error, but thats outside this PR.